### PR TITLE
Foodtype update

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

After eating a burrito and noticing the odd addition of the meat taste, I took a dive into the food folder and it hurt my poor fragile brain. I gave a lot of things updated tags or tastes. I think I got most of it.

## Why It's Good For The Game

Lizards can not eat nachos now, please bully them while having Mexican themed parties.
On the bright side, they can now eat potato and sweet potato. 

## Changelog
:cl:
fix: fixed inaccurate foodtypes/taste tags
/:cl:

### CODE REVIEW

File: code/game/objects/items/food/spaghetti.dm [hunk 1]
```diff
@@ -98,4 +100,4 @@
 	food_reagents = list(/datum/reagent/consumable/nutriment = 9, /datum/reagent/consumable/nutriment/vitamin = 2)
 	microwaved_type = null
 	tastes = list("noodle" = 1, "butter" = 1)
-	foodtypes = GRAIN | DAIRY
+	foodtypes = GRAIN | DAIRY
```
Comment (Rohesie) [inline code/game/objects/items/food/spaghetti.dm:103]:
```suggestion
	foodtypes = GRAIN | DAIRY
```
Typo, I assume.

File: code/game/objects/items/food/pizza.dm [hunk 2]
```diff
@@ -158,7 +158,7 @@
 	desc = "Deliciously sassy."
 	icon_state = "sassysagepizzaslice"
 	tastes = list("crust" = 1, "tomato" = 1, "cheese" = 1, "meat" = 1)
-	foodtypes = GRAIN | VEGETABLES | DAIRY
+	foodtypes = GRAIN | VEGETABLES | DAIRY | MEAT
```
Comment (Rohesie) [inline code/game/objects/items/food/pizza.dm:161]:
```suggestion
	foodtypes = GRAIN | VEGETABLES | DAIRY | MEAT
```

File: code/game/objects/items/food/pizza.dm [hunk 3]
```diff
@@ -149,7 +149,7 @@
 	icon_state = "sassysagepizza"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 20, /datum/reagent/consumable/nutriment/protein = 15, /datum/reagent/consumable/tomatojuice = 6, /datum/reagent/consumable/nutriment/vitamin = 6)
 	tastes = list("crust" = 1, "tomato" = 1, "cheese" = 1, "meat" = 1)
-	foodtypes = GRAIN | VEGETABLES | DAIRY
+	foodtypes = GRAIN | VEGETABLES | DAIRY | MEAT
```
Comment (Rohesie) [inline code/game/objects/items/food/pizza.dm:152]:
```suggestion
	foodtypes = GRAIN | VEGETABLES | DAIRY | MEAT
```

File: code/game/objects/items/food/misc.dm [hunk 4]
```diff
@@ -122,7 +122,7 @@
 
 	food_reagents = list(/datum/reagent/consumable/nutriment = 4)
 	tastes = list("fries" = 3, "salt" = 1)
-	foodtypes = VEGETABLES | GRAIN | FRIED
+	foodtypes = VEGETABLES | FRIED
```
Comment (Rohesie) [inline code/game/objects/items/food/misc.dm:125]:
```suggestion
	foodtypes = VEGETABLES | FRIED
```

File: code/game/objects/items/food/meat.dm [hunk 5]
```diff
@@ -1276,6 +1276,6 @@
 	icon_state = "full_english"
 	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 8, /datum/reagent/consumable/nutriment/vitamin = 4)
 	tastes = list("sausage" = 1, "bacon" = 1, "egg" = 1, "tomato" = 1, "mushrooms" = 1, "bread" = 1, "beans" = 1)
-	foodtypes = MEAT | VEGETABLES | GRAIN | BREAKFAST
+	foodtypes = MEAT | VEGETABLES | GRAIN | BREAKFAST
```
Comment (Rohesie) [inline code/game/objects/items/food/meat.dm:1279]:
```suggestion
	foodtypes = MEAT | VEGETABLES | GRAIN | BREAKFAST
```

File: code/game/objects/items/food/frozen.dm [hunk 6]
```diff
@@ -194,20 +198,23 @@
...
 	tastes = list("ice" = 1, "water" = 1, "flowers" = 5, "sweetness" = 5, "wax" = 1)
+	foodtypes = SUGAR 
 
 /obj/item/food/snowcones/rainbow
 	name = "rainbow snowcone"
 	desc = "A very colorful snowball in a paper cup."
 	icon_state = "rainbow_sc"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/laughter = 25, /datum/reagent/water = 11)
 	tastes = list("ice" = 1, "water" = 1, "sunlight" = 5, "light" = 5, "slime" = 5, "paint" = 3, "clouds" = 3)
+	foodtypes = SUGAR
```
Comment (Rohesie) [inline code/game/objects/items/food/frozen.dm:217]:
```suggestion
	foodtypes = SUGAR
```

File: code/game/objects/items/food/frozen.dm [hunk 7]
```diff
@@ -194,20 +198,23 @@
...
 	tastes = list("ice" = 1, "water" = 1, "valid" = 5, "salt" = 5, "wats" = 5)
+	foodtypes = SUGAR 
 
 /obj/item/food/snowcones/honey
 	name = "honey snowcone"
 	desc = "Honey drizzled over a snowball in a paper cup."
 	icon_state = "amber_sc"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/honey = 5, /datum/reagent/water = 11)
 	tastes = list("ice" = 1, "water" = 1, "flowers" = 5, "sweetness" = 5, "wax" = 1)
+	foodtypes = SUGAR
```
Comment (Rohesie) [inline code/game/objects/items/food/frozen.dm:209]:
```suggestion
	foodtypes = SUGAR
```

File: code/game/objects/items/food/frozen.dm [hunk 8]
```diff
@@ -194,20 +198,23 @@
 	icon_state = "pwrgame_sc"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/pwr_game = 5, /datum/reagent/water = 11)
 	tastes = list("ice" = 1, "water" = 1, "valid" = 5, "salt" = 5, "wats" = 5)
+	foodtypes = SUGAR
```
Comment (Rohesie) [inline code/game/objects/items/food/frozen.dm:201]:
```suggestion
	foodtypes = SUGAR
```

File: code/game/objects/items/food/frozen.dm [hunk 9]
```diff
@@ -165,27 +165,31 @@
...
 	tastes = list("ice" = 1, "water" = 1, "cola" = 5)
+	foodtypes = SUGAR 
 
 /obj/item/food/snowcones/spacemountainwind
 	name = "Space Mountain Wind snowcone"
 	desc = "Space Mountain Wind drizzled over a snowball in a paper cup."
 	icon_state = "mountainwind_sc"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/spacemountainwind = 5, /datum/reagent/water = 11)
 	tastes = list("ice" = 1, "water" = 1, "mountain wind" = 5)
+	foodtypes = SUGAR
```
Comment (Rohesie) [inline code/game/objects/items/food/frozen.dm:192]:
```suggestion
	foodtypes = SUGAR
```

File: code/game/objects/items/food/frozen.dm [hunk 10]
```diff
@@ -165,27 +165,31 @@
...
 	tastes = list("ice" = 1, "water" = 1, "jokes" = 5, "brainfreeze" = 5, "joy" = 5)
+	foodtypes = SUGAR | FRUIT 
 
 /obj/item/food/snowcones/soda
 	name = "space cola snowcone"
 	desc = "Space Cola drizzled over a snowball in a paper cup."
 	icon_state = "soda_sc"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/space_cola = 5, /datum/reagent/water = 11)
 	tastes = list("ice" = 1, "water" = 1, "cola" = 5)
+	foodtypes = SUGAR
```
Comment (Rohesie) [inline code/game/objects/items/food/frozen.dm:184]:
```suggestion
	foodtypes = SUGAR
```

File: code/game/objects/items/food/frozen.dm [hunk 11]
```diff
@@ -165,27 +165,31 @@
...
 	tastes = list("ice" = 1, "water" = 1, "nothing" = 5)
+	foodtypes = SUGAR 
 
 /obj/item/food/snowcones/clown
 	name = "clown snowcone"
 	desc = "Laughter drizzled over a snowball in a paper cup."
 	icon_state = "clown_sc"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/laughter = 5, /datum/reagent/water = 11)
 	tastes = list("ice" = 1, "water" = 1, "jokes" = 5, "brainfreeze" = 5, "joy" = 5)
+	foodtypes = SUGAR | FRUIT
```
Comment (Rohesie) [inline code/game/objects/items/food/frozen.dm:176]:
```suggestion
	foodtypes = SUGAR | FRUIT
```

File: code/game/objects/items/food/frozen.dm [hunk 12]
```diff
@@ -165,27 +165,31 @@
 	icon_state = "mime_sc"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/nothing = 5, /datum/reagent/water = 11)
 	tastes = list("ice" = 1, "water" = 1, "nothing" = 5)
+	foodtypes = SUGAR
```
Comment (Rohesie) [inline code/game/objects/items/food/frozen.dm:168]:
```suggestion
	foodtypes = SUGAR
```

**Comment by carshalash** [comment]
Oops.

**Pull request merged**